projects/plugins/boost/app/modules/cache/cache.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+class Boost_Cache {
+	public $key         = false;
+	public $request_uri = false;
```

Comment:
IMHO `$key` ought to be private, as even child classes will need to use `->key()` to get it. `$request_uri` ought to be `protected`.

Reply:
You're right, thanks!